gradle/plugins/src/main/kotlin/com/saveourtool/save/buildutils/save-cli-configuration.gradle.kts

```diff
@@ -51,13 +51,11 @@ dependencies {
     )
 }
 
-// todo: this logic is duplicated between agent and frontend, can be moved to a shared plugin in gradle/plugins
```

Comment:
now it's not duplicated or what?

Reply:
_frontend_ needs only `SAVE_CLOUD_VERSION` (as `SAVE_VERSION`), _backend_ and _sandbox_ need `SAVE_CORE_VERSION`.
_agent_ needs `AGENT_VERSION` (it's `SAVE_CLOUD_VERSION`).
Now we have similar task between _frontend_ and _agent_ -- will try to combine

Reply:
done